app/src/main/java/ch/epfl/skysync/screens/admin/AdminCalendar.kt

```diff
@@ -0,0 +1,92 @@
+package ch.epfl.skysync.screens.admin
+
+import android.util.Log
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Tab
+import androidx.compose.material3.TabRow
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import ch.epfl.skysync.components.AvailabilityCalendar
+import ch.epfl.skysync.components.FlightCalendar
+import ch.epfl.skysync.navigation.AdminBottomBar
+import ch.epfl.skysync.navigation.Route
+import ch.epfl.skysync.ui.theme.lightGray
+import ch.epfl.skysync.viewmodel.CalendarViewModel
+
+@Composable
+fun AdminCalendarScreen(

```

Comment:
I don't know exactly the differences between the admin and crew/pilot version of the calendar screen,
* What are the UI difference, are there easy to modularize ? (My point is whether or not we can make a single calendar component and just pass different parameters to it)
* The view models will be different, but is the front end logic the same ? (That is, do we take the same callbacks ?)